File: protivCenter/protiv-frontend/src/app/model/natural-focus-of-plague.ts
import {Report} from './report';
import {Station} from './station';

export interface NaturalFocusOfPlague extends Report{
  id: number;
  station: Station;
  year: number;
  month: number;
  area_fiz: number;
  area_oper: number;
  nocit: number;
  blox: number;
  orud_pole: number;
  orud_punkt: number;
  enter: number;
  bac_nosit: number;
  krov_all: number;
  krov_blox: number;
}

File: protivCenter/protiv/src/main/java/com/example/demo/entity/reports/NaturalFocusOfPlague.java
package com.example.demo.entity.reports;

import com.example.demo.entity.ReportBase;
import lombok.Data;
import javax.persistence.Entity;

@Entity
@Data
public class NaturalFocusOfPlague extends ReportBase {
    private double area_fiz;
    private double area_oper;
    private double nocit;
    private double blox;
    private int orud_pole;
    private int orud_punkt;
    private int enter;
    private double bac_nosit;
    private double krov_all;
    private double krov_blox;
}
